src/aof.c

```diff
@@ -1073,7 +1073,8 @@ void flushAppendOnlyFile(int force) {
              * (because there's no reason, from the AOF POV, to call fsync) and then WAITAOF may wait on
```

Comment:
I notice in this pr, you update comment as well, pls update the above line master_repl_offset to primary_repl_offset